ios/brave-ios/scripts/xcode_scheme_preaction.sh

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# This script is run by Xcode and assumes Xcode build settings are injected
+# into the environment.
+
+# $ACTION is empty for Xcode builds unfortunately
+# $RUN_CLANG_STATIC_ANALYZER is NO for builds, YES for cleans
+if [[ $RUN_CLANG_STATIC_ANALYZER = "NO" ]]; then
+  # Do not inject Xcode build configs into the GN build
+  env -i PATH="$PATH" python3 \
```

Comment:
Fixed here: https://github.com/brave/brave-core/pull/21553/commits/c30d30d5d06854dde71c9b472a3f7a74f51c904b I also attempt to fixup the PATH for users who install node/npm via brew on arm64 machines (since brew prefix on those machines are `/opt/homebrew`)